modules/using-certificates-with-data-science-pipelines.adoc

```diff
@@ -0,0 +1,85 @@
+:_module-type: PROCEDURE
+
+[id='using-certificates-with-data-science-pipelines_{context}']
+= Using certificates with data science pipelines 
+
+== Providing a CA bundle only for data science pipelines
+
+Perform the following steps to provide a Certificate Authority (CA) bundle just for data science pipelines.
+
```

Comment:
Any prerequisites?

Reply:
I was told no, just the CA bundle they want to add.